pkg/cmd/admin/network/project_options.go

```diff
@@ -182,5 +162,21 @@ func (p *ProjectOptions) UpdatePodNetwork(nsName string, action sdnapi.PodNetwor
 	}
 
 	// Validate SDN controller applied or rejected the intent
-	return p.validateNetNamespace(netns)
+	backoff := wait.Backoff{
+		Steps:    30,
+		Duration: 500 * time.Millisecond,
+		Factor:   1.1,
```

Comment:
The old code waited 10 seconds total; if I'm doing the math right, this would come to about 80 seconds. Steps=12 would give you about 10 seconds again.